### PR TITLE
Create helm chart for wallarm-sidecar 

### PR DESCRIPTION
**Pull request by xDmitriev**

### CODE REVIEW

File: templates/webhook-configuration.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  name: {{ include "wallarm-sidecar.name" . }}-webhook
+webhooks:
+  - name: injector.sidecar.wallarm.io
+    matchPolicy: Equivalent
+    rules:
+      - apiGroups: ["*"]
+        apiVersions: ["v1"]
+        operations:
+          - CREATE
+          - UPDATE
```
Comment (MrEcco) [inline templates/webhook-configuration.yaml]: Do you properly check "UPDATE" verb?

```bash
kubectl label pods pod-name wallarm-sidecar=enabled
kubectl annotate pods pod-name sidecar.wallarm.io/wallarm-mode=block
```

File: templates/_helpers.tpl [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+runAsUser: {{ .Values.postanalytics.runAsUser }}
+allowPrivilegeEscalation: {{ .Values.postanalytics.allowPrivilegeEscalation }}
+{{- end }}
+{{- end -}}
+
+{{- define "wallarm-sidecar.postanalytics.wallarmApiEnv" -}}
+- name: WALLARM_API_HOST
+  value: {{ .Values.wallarmApi.host | default "api.wallarm.com" }}
+- name: WALLARM_API_PORT
+  value: {{ .Values.wallarmApi.port | default "444" | quote }}
```
Comment (MrEcco) [inline templates/_helpers.tpl]: Since 4.0 release default port for Wallarm API is 443 :)

Comment (xDmitriev) [reply templates/_helpers.tpl]: Fixed. Now tt comes from default values only, where it is set to 443.

File: templates/webhook-configuration.yaml (hunk 1 above)
Comment (xDmitriev) [reply templates/webhook-configuration.yaml]: Removed UPDATE operation

File: test/integration_test.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+        namespace_created = helpers.create_namespace(namespace)
+        assert namespace_created.returncode == 0, namespace_created.stderr
+
+        create = helpers.create(config_path, namespace)
+        assert create.returncode == 0, create.stderr
+
+        wait = helpers.wait(namespace)
+        assert wait.returncode == 0, wait.stderr
+
+        sleep(5)
```
Comment (ep4sh) [inline test/integration_test.py]: Could you please explain why do we wait/sleep here, please wrap it with comment if needed

File: values.yaml [hunk 4]
```diff
@@ -0,0 +1,303 @@
...
+  annotations: {}
+  # -- Labels to be added to admission webhook resources
+  labels: {}
+  # -- Admission webhook failure policy
+  failurePolicy: Fail
+  image:
+    # -- Image registry
+    registry: quay.io
+    # -- Image repository
+    image: dmitriev/sidecar-injector
```
Comment (ep4sh) [inline values.yaml:257]: should it be moved to the official repo at this moment?

**Review by ep4sh — CHANGES_REQUESTED** [review]

File: .github/workflows/integrationTest.yml [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+on:
+  push:
+    branches: [ "main", "helm-chart", "github-actions" ]
+    tags:
+      - '**'
+#  pull_request:
+#    branches: [ "main" ]
+#
+#
```
Comment (ep4sh) [inline .github/workflows/integrationTest.yml]: plz remove the comments if redundant

File: .github/workflows/testArtifacts.yml [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+on:
+  push:
+    branches: [ "main", "helm-chart", "github-actions" ]
+    tags:
+      - '**'
+#  pull_request:
+#    branches: [ "main" ]
+#
+#
```
Comment (ep4sh) [inline .github/workflows/testArtifacts.yml]: plz remove the comments if redundant

**Review by ep4sh — CHANGES_REQUESTED** [review]

File: values.yaml (hunk 4 above)
Comment (xDmitriev) [reply values.yaml:257]: It's planned for the near future

**Pull request merged**